src/Microsoft.OData.Client/MaterializeFromObject.cs

```diff
@@ -1,5 +1,5 @@
 ﻿//---------------------------------------------------------------------
-// <copyright file="MaterializeFromAtom.cs" company="Microsoft">
+// <copyright file="MaterializeFromObject.cs" company="Microsoft">

```

Comment:
Any reason why the filename doesn't have the same name as the class?

Reply:
Not really. It was like that before. I can change it.